### PR TITLE
Add ability to change hostname/port

### PR DESCRIPTION
**Pull request by rschick**

### CODE REVIEW

**Review by dlockhart — COMMENTED** [review]
I think you can do this without introducing environment variables? Like with an NPM script you can pass extra arguments to the underlying script using `--`.

So for `es-dev-server` you could go:

```shell
npm start -- --port=1234 --hostname=foo
```

**Comment by rschick** [comment]
I couldn't get it to work that way, but `$npm_config_` works.. (TIL!)

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "publish:autotag": "node ./cli/autotag.js",
     "publish:cdn": "frau-publisher -m lib -t bsi -f \"./build/**/*\" --versionvar TRAVIS_TAG",
     "start": "npm run start:dev",
-    "start:dev": "concurrently --kill-others \"es-dev-server --config es-dev-server-dev.config.js\" \"rollup -w -c ./rollup/rollup.config.js\"",
+    "start:dev": "concurrently --kill-others \"es-dev-server --config es-dev-server-dev.config.js --hostname $npm_config_hostname --port $npm_config_port\" \"rollup -w -c ./rollup/rollup.config.js\"",
```
Comment (dlockhart) [inline package.json:14]: Running this without passing anything changes the port to 8000 for me.

**Comment by rschick** [comment]
Good catch, I tested it and didn't notice the port was wrong. Updated with a cleaner approach.

**Review by dlockhart — APPROVED** [review]
👍

**Pull request merged**